docs/installation-guide.md

```diff
@@ -92,77 +96,87 @@ To get started with CarbonData : [Quick Start](quick-start-guide.md), [DDL Opera
 
-```
+
+6. Add the following properties in `<SPARK_HOME>/conf/carbon.properties`:
+
+   | Property | Required | Description | Example | Default Value |
+   |----------------------|----------|----------------------------------------------------------------------------------------|-------------------------------------|---------------|
+   | carbon.storelocation | NO | Location where CarbonData will create the store and write the data in its own format. | hdfs://HOSTNAME:PORT/Opt/CarbonStore | Propose to set HDFS directory|
+   | carbon.kettle.home | YES | Path that will be used by CarbonData internally to create graph for loading the data. | carbondata.tar.gz/carbonlib/carbonplugins |  |
+
+
+7. Verify the installation.
+
+   ```
      ./bin/spark-shell --master yarn-client --driver-memory 1g
      --executor-cores 2 --executor-memory 2G
-```
-  NOTE: Make sure you have permissions for CarbonData JARs and files through which driver and executor will start.
+   ```
+  **NOTE**: Make sure you have permissions for CarbonData JARs and files through which driver and executor will start.
 
   Getting started with CarbonData : [Quick Start](quick-start-guide.md), [DDL Operations on CarbonData](ddl-operation-on-carbondata.md)
 
 ## Query Execution Using CarbonData Thrift Server
 
-### Starting CarbonData Thrift Server
+### Starting CarbonData Thrift Server.
 
-   a. cd ``<SPARK_HOME>``
+   a. cd `<SPARK_HOME>`
 
    b. Run the following command to start the CarbonData thrift server.
 
-```
-./bin/spark-submit --conf spark.sql.hive.thriftServer.singleSession=true
---class org.apache.carbondata.spark.thriftserver.CarbonThriftServer
-$SPARK_HOME/carbonlib/$CARBON_ASSEMBLY_JAR <carbon_store_path>
-```
+   ```
+   ./bin/spark-submit --conf spark.sql.hive.thriftServer.singleSession=true
+   --class org.apache.carbondata.spark.thriftserver.CarbonThriftServer
+   <SPARK_HOME>/carbonlib/$CARBON_ASSEMBLY_JAR <carbon_store_path>
+   ```
 
-| Parameter | Description | Example |
-|---------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------|
-| CARBON_ASSEMBLY_JAR | CarbonData assembly jar name present in the ``"<SPARK_HOME>"/carbonlib/`` folder. | carbondata_2.10-0.1.0-incubating-SNAPSHOT-shade-hadoop2.7.2.jar |
-| carbon_store_path | This is a parameter to the CarbonThriftServer class. This a HDFS path where CarbonData files will be kept. Strongly Recommended to put same as carbon.storelocation parameter of carbon.properties. | ``hdfs//<host_name>:54310/user/hive/warehouse/carbon.store`` |
+   | Parameter | Description | Example |
```

Comment:
The syntax of this table may be not correct, please click `view` button which is locate at top-right corner for check & review.